src/module.h

```diff
@@ -211,6 +211,7 @@ int TerminateModuleForkChild(int child_pid, int wait);
 ssize_t rdbSaveModulesAux(rio *rdb, int when);
 int moduleAllDatatypesHandleErrors(void);
 int moduleAllModulesHandleReplAsyncLoad(void);
+int moduleVerifyAllModulesAllowAtomicSlotMigrationOrReply(client *c);
```

Comment:
I would prefer it to be
```suggestion
int verifyAllModulesAllowAtomicSlotMigration(void);
```

Reply:
May I ask why? Are prefixes like these not useful?

Reply:
Yeah, I don't see any value.

Reply:
I still think unit name in the method name seems redundant to me. If people actually benefit from this, I don't want to push this further.

AWS `amz` thing is to indicate internal code i.e. definitely helpful.